.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug report
+description: Something isn't working correctly.
+labels: '0.kind: bug'
+
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the bug.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Metadata
+      description: Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result here.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Notify maintainers
+      description: |
+        Please @ people who are in the `meta.maintainers` list of the offending package or module.
+        If in doubt, check `git blame` for whoever last touched something.
+    validations:
+      required: true
```

Comment:
I like the idea of requiring a ping of the maintainer but many newbies will have no idea who to ping or how to follow those steps and probably give up.

Not worth it IMO.

Reply:
A better policy might be to achieve maintainer/people who know about the topic at hand is to use the assignee system and have maintainers assign them based on experience (I know this person maintains/knows a lot about about *x*) for this but that would be a topic for an RFC I think.

Reply:
Perhaps we should suggest using [search.nixos.org](https://search.nixos.org/packages?channel=unstable&show=hello&from=0&size=50&sort=relevance&type=packages&query=hello) as an easier way to find maintainers. The link on each maintainer's name leads to their GitHub profile; we could even get the website modified so that their GitHub usernames are displayed directly.

As you said, some packages don't actually have any maintainers, so this field probably shouldn't be strictly required.